Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -371,6 +371,11 @@ if (WIN32)
 endif()
 
 if (MSVC)
+  # Disable some warnings
+  target_compile_options(dolphin-emu PRIVATE
+    /wd5054 # operator '+': deprecated between enumerations of different types (types defined in Qt headers)
```

Comment:
Why is this needed when you already disable all warnings in external code?

Reply:
Fwiw shouldn’t be required anymore as qt fixed their code